pkg/pillar/types/global.go

```diff
@@ -186,6 +186,8 @@ const (
 	AllowAppVnc GlobalSettingKey = "app.allow.vnc"
 	// EveMemoryLimitInBytes global setting key
 	EveMemoryLimitInBytes GlobalSettingKey = "memory.eve.limit.bytes"
+	// How much memory overhead is allowed for VMM needs
+	VmmMemoryLimitInMiB GlobalSettingKey = "memory.vmm.limit.MiB"
```

Comment:
Having the type as part of the name is a good idea. But I thinkl this is the first one which isn't lower case. Can you check if zedcloud and EVE handle that, and whether or not the lookup is case sensitive? (Whether or not one can set memory.vmm.limit.mib.)

Reply:
Ok